deployment/helpers/deployment-helpers.js

```diff
@@ -37,7 +37,8 @@ async function deployPolygonZkEVMDeployer(deployerAddress, signer) {
     const zkEVMDeployerAddress = ethers.utils.getContractAddress(resultTransaction);
     if (await signer.provider.getCode(zkEVMDeployerAddress) !== '0x') {
         const zkEVMDeployerContract = PolgonZKEVMDeployerFactory.attach(zkEVMDeployerAddress);
-        expect(await zkEVMDeployerContract.owner()).to.be.equal(signer.address);
+        const ownerAddress = await zkEVMDeployerContract.owner();
+        expect(ownerAddress.toLowerCase()).to.be.equal(signer.address);
```

Comment:
just curious, is signer.address always in lower case?

Reply:
not sure, and will add `.toLowerCase()` for it.